projects/1.19/assets/random-enchants-2/randomenchants2/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "enchantment.randomenchants2.solar_enchant": "日光修补",
+  "enchantment.randomenchants2.solar_enchant.desc" : "放在主副手上的装备在光源附近下会缓慢恢复",
+  "enchantment.randomenchants2.obsidian_buster": "黑曜石克星",
+  "enchantment.randomenchants2.obsidian_buster.desc": "提高黑曜石的挖掘速度",
+  "enchantment.randomenchants2.randomness": "随机掉落",
+  "enchantment.randomenchants2.randomness.desc": "开采任意方块时会掉落随机物品",
+  "enchantment.randomenchants2.magnetic" : "磁性",
+  "enchantment.randomenchants2.magnetic.desc" : "装备会将把开采的方块/掉落物传送到您的背包中",
+  "enchantment.randomenchants2.stone_lover" : "爱石",
+  "enchantment.randomenchants2.stone_lover.desc" : "开采非圆石的其他石头有80％的几率恢复耐久",
+  "enchantment.randomenchants2.eternal" : "亘古永恒",
+  "enchantment.randomenchants2.eternal.desc" : "具有此附魔的装备不再因时间原因而消失，岩浆也不可以烧掉",
+  "enchantment.randomenchants2.dungeoneering" : "寻宝者",
+  "enchantment.randomenchants2.dungeoneering.desc" : "在杀死生物时有几率掉落稀有物品",
+  "enchantment.randomenchants2.grappling" : "抓钩",
+  "enchantment.randomenchants2.grappling.desc" : "鱼钩钩到方块后，右键可以将玩家扯到该位置附近，当你勾到比自己高不到5个方块的方块右键则会飞起",

```

```suggestion
  "enchantment.randomenchants2.grappling.desc" : "鱼钩钩到方块后，右击可以将玩家扯到该位置附近，当你勾到比自己高不到5个方块的方块右键则会飞起",
```
前后主语称呼不一致（玩家/你/自己），需要修改
右键不可以成为主谓结构，右键什么？右键单击？右键长按？